python/gtsam/examples/CombinedImuFactorExample.py

```diff
@@ -0,0 +1,251 @@
+"""
```

Comment:
Is this a brand new python example?

Reply:
Yup. I wanted to add one so we can do a quick comparison with the ImuFactor example.